game-core/src/main/java/games/strategy/engine/chat/PlayerChatRenderer.java

```diff
@@ -45,24 +44,28 @@ public PlayerChatRenderer(final IGame game, final UiContext uiContext) {
   @Override
   public Component getListCellRendererComponent(final JList<?> list, final Object value, final int index,
       final boolean isSelected, final boolean cellHasFocus) {
-    final List<Icon> icons = iconMap.get(value.toString());
+    final INode node = (INode) value;
+    final List<Icon> icons = iconMap.get(node.toString());
     if (icons != null) {
-      super.getListCellRendererComponent(list, ((INode) value).getName(), index, isSelected, cellHasFocus);
+      super.getListCellRendererComponent(list, getNodeLabel(node), index, isSelected, cellHasFocus);
       setHorizontalTextPosition(SwingConstants.LEFT);
       setIcon(new CompositeIcon(icons));
     } else {
-      final StringBuilder sb = new StringBuilder(((INode) value).getName());
-      final Set<String> players = playerMap.get(value.toString());
-      if (players != null && !players.isEmpty()) {
-        sb.append(" (");
-        sb.append(Joiner.on(", ").join(players));
-        sb.append(")");
-      }
-      super.getListCellRendererComponent(list, sb.toString(), index, isSelected, cellHasFocus);
+      super.getListCellRendererComponent(list, getNodeLabelWithPlayers(node), index, isSelected, cellHasFocus);
     }
     return this;
   }
 
+  private static String getNodeLabel(final INode node) {
+    return node.getName();
+  }
```

Comment:
I don't think this one-line wrapping mechanism that doesn't even check for null is worth it 😅

Reply:
639042caf